compiler/rustc_trait_selection/src/traits/error_reporting/mod.rs

```diff
@@ -2348,12 +2341,12 @@ impl<'tcx> InferCtxtPrivExt<'tcx> for TypeErrCtxt<'_, 'tcx> {
                                 predicate.to_opt_poly_trait_pred().unwrap(),
                             );
                             if impl_candidates.len() < 10 {
-                                let hir =
-                                    self.tcx.hir().local_def_id_to_hir_id(obligation.cause.body_id);
                                 self.report_similar_impl_candidates(
                                     impl_candidates,
                                     trait_ref,
-                                    body_id.map(|id| id.hir_id).unwrap_or(hir),
+                                    body_def_id
+                                        .map(|id| id)
```

Comment:
This doesn't do anything :smile:

Reply:
lol, clearly I wasn't even thinking and just removed `.hir_id`